_data/simple-icons.json

```
@@ -3694,6 +3694,11 @@
                 ]
             }
         },
+        {
+            "title": "Grab",
+            "hex": "1AAE48",
+            "source": "https://en.wikipedia.org/wiki/Grab_(company)"
```

As per [our source guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), this should be: 

```suggestion
            "source": "https://en.wikipedia.org/wiki/File:Grab_(application)_logo.svg)"
```